src/components/Tokens/TokenDetails/ActivitySection.tsx

```diff
@@ -0,0 +1,49 @@
+import { Trans } from '@lingui/macro'
+import { ChainId, Token } from '@uniswap/sdk-core'
+import Row from 'components/Row'
+import { useState } from 'react'
+import styled from 'styled-components'
+import { ClickableStyle } from 'theme/components'
+import { ThemedText } from 'theme/components/text'
+
+import { TransactionsTable } from './TransactionsTable'
+
+const Container = styled.div`
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+`
+const Tab = styled(ThemedText.BodyPrimary)<{ isActive?: boolean }>`
+  cursor: pointer;
+  font-size: 24px !important;
```

Comment:
in these cases where you end up overwriting the main styles of the `ThemedText` I find it preferable to extend the base `Text` class to avoid `!important` override